scripts/auto-stop-idle/on-start.sh

```diff
@@ -9,7 +9,8 @@ set -e
 #
 # Note that this script will fail if either condition is not met
 #   1. Ensure the Notebook Instance has internet connectivity to fetch the example config
-#   2. Ensure the Notebook Instance execution role permissions to SageMaker:StopNotebookInstance to stop the notebook
+#   2. Ensure the Notebook Instance execution role permissions to SageMaker:StopNotebookInstance to stop the notebook 
+#       and SageMaker:StopNotebookInstance to describe the notebook.
```

Comment:
Don't think you need this change.

Reply:
See above comments.

Reply:
Do you mind fixing the typo here? should be ```SageMaker:DescribeNotebookInstance```

Reply:
Fixed in latest commit.